tests/test_ec2/test_security_groups.py

```diff
@@ -2332,3 +2379,40 @@ def test_get_groups_by_ippermissions_group_id_filter_across_vpcs():
     security_groups = list(security_groups)
     assert len(security_groups) == 1
     assert security_groups[0].group_id == sg1.group_id
+
+
+@mock_ec2
+def test_filter_group_name():
+    """
+    this filter is an exact match, not a glob
+    """
+    ec2r = boto3.resource("ec2", region_name="us-west-1")
+    vpc = ec2r.create_vpc(CidrBlock="10.250.1.0/16")
+
+    uniq_sg_name_prefix = str(uuid4())[0:6]
+    sg1 = vpc.create_security_group(
+        Description="A Described Description Descriptor",
+        GroupName=f"{uniq_sg_name_prefix}-test-1",
+    )
+    vpc.create_security_group(
+        Description="Another Description That Awes The Human Mind",
+        GroupName=f"{uniq_sg_name_prefix}-test-12",
+    )
+    vpc.create_security_group(
+        Description="Yet Another Descriptive Description",
+        GroupName=f"{uniq_sg_name_prefix}-test-13",
+    )
+    vpc.create_security_group(
+        Description="Such Description Much Described",
+        GroupName=f"{uniq_sg_name_prefix}-test-14",
+    )
+
+    filter_to_match_group_1 = {
+        "Name": "group-name",
+        "Values": [sg1.group_name],
+    }
+    security_groups = ec2r.security_groups.filter(Filters=[filter_to_match_group_1])
+
+    security_groups = list(security_groups)
+    assert len(security_groups) == 1
+    assert security_groups[0].group_name == "test-1"
```

Comment:
```suggestion
    assert security_groups[0].group_name == sg1.group_name
```

Reply:
derp.

Reply:
appreciate that @bblommers! apologies for missing that there.